pkg/core/mempool/mem_pool.go

```diff
@@ -232,9 +286,16 @@ func (mp *Pool) RemoveStale(isOK func(*transaction.Transaction) bool, feer Feer)
 	// because items are iterated one-by-one in increasing order.
 	newVerifiedTxes := mp.verifiedTxes[:0]
 	mp.fees = make(map[util.Uint160]utilityBalanceAndFees) // it'd be nice to reuse existing map, but we can't easily clear it
+	mp.conflicts = make(map[util.Uint256][]util.Uint256)
```

Comment:
`if feer.P2PSigExtensionsEnabled()` for it? It's a micro-optimization for non-extended protocol.